src/app/containers/PageViewAnalytics/labelHelpers/index.test.js

```diff
@@ -214,11 +214,20 @@ describe('getCurrentTime', () => {
 
   it('should return hours, mins and seconds joined by "x"', () => {
     const date = new Date(946729425000); // 12:23:45
+    const currentTimeZoneOffsetInHours = date.getTimezoneOffset() / 60;
```

Comment:
Not convinced this will work in timezones that differ on 30 mins rather than an hour. But it's a fix that enables Andela to carry on working. Can improve the logic to handle minutes if needs be in the future

Reply:
To be the clear, the production code works in all cases. It's purely the unit tests that were failing

Reply:
Timezone was a overly complicated fix, just should mock the values in the test 😄